### DISCUSSION TITLE
Framed cobblestone and framed spruce are not available in the "minimum stock" so you can't maintain mines.

### DISCUSSION THREAD

**Discussion by servireettueri** [ldtteam/minecolonies#8852] [General]
It's also not in GEI (though I know that's another mod so I imagine I would need to post there?). Only "framed dark oak plank" and "Framed glowstone" are available. Only reason I was able to craft them was thanks to a reddit post as even the wiki has no info on these blocks. If i try to load up the miner with a few stacks of resources then the just get taken to the warehouse so I have to come back every few minutes and manually return the blocks only for the to get taken again.

**Comment by MotionlessTrain** [comment]
There are millions of possibilities. That is why they don't show up in those menus, that would give performance issues (and very hard to scroll though). If you have one of those items in your inventory, it shows up in the minimum stock list.
The wiki has information about those blocks and the architects cutter in "Dependencies" -> "Domum Ornamentum"

**Reply by servireettueri** [reply]
> There are millions of possibilities. That is why they don't show up in those menus, that would give performance issues (and very hard to scroll though). If you have one of those items in your inventory, it shows up in the minimum stock list. The wiki has information about those blocks and the architects cutter in "Dependencies" -> "Domum Ornamentum"

THANKS YOU. I didn't know about the inventory thing,

**Comment by Dennis1108** [comment]
Hello there, i have this problem. But i have the item in my inventory... but still not showing it to me.... i wrote in discord as Hyp0thalamus, playing on the actual patreon server.

**Comment by Dennis1108** [comment]
![Screenshot 2025-02-15 101500](https://github.com/user-attachments/assets/55ee9546-2890-46a1-9114-80dff69831ce)
![Screenshot 2025-02-15 101536](https://github.com/user-attachments/assets/db9d5c52-e1c4-4ed5-a4f1-ad735c7afeb1)
![Screenshot 2025-02-15 202459](https://github.com/user-attachments/assets/38d75cd1-1d10-4fde-a070-35814701f5b0)
![Screenshot 2025-02-15 202539](https://github.com/user-attachments/assets/db86b49d-93c5-4b37-a904-609180e2bb75)
![Screenshot 2025-02-15 203220](https://github.com/user-attachments/assets/3451ecab-af51-4a2d-9479-24b89da05938)
![Screenshot 2025-02-15 203433](https://github.com/user-attachments/assets/ac00a453-a9a5-4854-9e09-04d656af0908)

**Comment by MixusMinimax** [comment]
in my experience, the items from your inventory show up in the list at the top if you don't type into the search bar.
